components/api_version/Cargo.toml

```diff
@@ -15,8 +15,6 @@ engine_traits = { workspace = true }
 kvproto = { workspace = true }
 log_wrappers = { workspace = true }
 match-template = "0.0.1"
-thiserror = "1.0"
-tikv_alloc = { workspace = true }
```

Comment:
Will it cause this crate to not use tikv allocator?

Reply:
Nope, check the doc of `tikv_alloc`. This component is linked to `tikv` and it doesn't even write `extern crate tikv_alloc`. So simply removing the dependency doesn't have any difference.
```
//! Writing `extern crate tikv_alloc;` will link it to jemalloc when
//! appropriate. The TiKV library itself links to `tikv_alloc` to
//! ensure that any binary linking to it will use jemalloc.
//!
//! With few exceptions, _every binary and project in the TiKV workspace
//! should link (perhaps transitively) to tikv_alloc_. This is to ensure
//! that tests and benchmarks run with the production allocator. In other
//! words, binaries and projects that don't link to `tikv` should link
//! to `tikv_alloc`.
```